src/components/pre-playback-play-overlay/_pre-playback-play-overlay.scss

```diff
@@ -55,4 +55,9 @@
       padding: 15px;
     }
   }
+
```

Comment:
sass syntax should be hierarchy

Reply:
I added it under .player isn't hierarchy? what do you mean? should i separate it like:
&.overlay-active 
 .gui-area 
 .pre-playback-play-overlay